namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -212,6 +225,51 @@ class ConsulDtabStore(
         Future.Unit
       }
     }
-    Activity(run).stabilize
+    dtabStatus.putIfAbsent(ns, InstrumentedDtab(run, pollState))
+    Activity(run.underlying).stabilize
   }
+
+  val handlerPrefix = root.show.drop(1) // drop leading "/"
+
+  override def adminHandlers: Seq[Admin.Handler] = Seq(
+    Admin.Handler(
+      s"/namer_dtabs_store/${handlerPrefix}.json",
```

Comment:
This results in a URL like `/namer_dtabs_store/namerd/dtabs.json`.  I we want to have a URL like `/namer_dtabs_store/namerd/default.json` where `default` is the name of dtab namespace.  This should be as simple as using `ns` instead of `root` here.

Reply:
wait nevermind, I think what you have here with using `root` is actually correct.  Ignore my previous comment.